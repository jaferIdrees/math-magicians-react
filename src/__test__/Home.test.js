import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Home component testing using snapshots', () => {
  it('Test Home component renders', () => {
    const tree = renderer
      .create(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
