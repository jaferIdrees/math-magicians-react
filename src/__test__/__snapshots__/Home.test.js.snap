// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing using snapshots Test Home component renders 1`] = `
<div
  className="homePageContainer"
>
  <h2
    className="homePageTitle"
  >
    Welcome to our page!
  </h2>
  <p
    className="homePageText"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</div>
`;
